src/lib/classes/lsp7-digital-asset.ts

```diff
@@ -50,28 +56,42 @@ export class LSP7DigitalAsset {
     digitalAssetDeploymentOptions: LSP7DigitalAssetDeploymentOptions,
     contractDeploymentOptions?: ContractDeploymentOptions
   ) {
+    const defaultBaseContractAddress: string | undefined =
+      contractDeploymentOptions?.libAddress ??
+      versions[this.options.chainId]?.contracts.LSP7Mintable?.versions[DEFAULT_CONTRACT_VERSION];
+
+    const deployProxy = contractDeploymentOptions?.deployProxy === false ? false : true;
```

Comment:
Why is the strict equal `===` necessary here?
Can't you just do

```typescript
const deployProxy = contractDeploymentOptions?.deployProxy
```

Reply:
Ah yeah indeed this is a bit tricky. We we want deployProxy to default to true here which means that if someone does not pass the object and `contractDeploymentOptions.deployProxy` is undefined, deployProxy should still be true. Ie we only want `deployProxy` to be false if a user specifically passed `contractDeploymentOptions?.deployProxy: false`